### ISSUE TITLE
Blog Styling needs to be cleaned up

### ISSUE THREAD

**Issue by akozlik** [jacquesfu/odevs-eleventy-starter#21]
* Left align blog posts
* "In the News" title is too close to the header. We need to determine our vertical spacing here.

![Screen Shot 2022-02-06 at 12 45 46 PM](https://user-images.githubusercontent.com/467951/152693888-26716776-9427-40b6-bf38-52fce0bd1036.png)

**Comment by jonkurtis** [comment]
I think the page layouts in general need to be cleaned up and de-duplicated. There seems to be a layout for every page that builds on the default layout but essentially just ads the title to an H1 and puts the content on the page. Instead we should have a general `page`, `post`, `event` layout which would include default spacing at the top of the page. I think it is better to add default spacing as margin top of the page layout instead of margin bottom to the header itself to give us the flexibility to add an element directly up against the header like a hero image.

**Comment by akozlik** [comment]
Yeah I definitely agree. We should pick a single layout per page type and stick wtih that. What do you need to move forward with that?

**Comment by jonkurtis** [comment]
Nothing I need. I will work on this.
